samples/subsys/usb/hid/src/main.c

```diff
@@ -81,36 +85,55 @@ int get_report_cb(struct usb_setup_packet *setup, s32_t *len,
 	return 0;
 }
 
+static void send_report(struct k_work *work)
+{
+	static u8_t report_1[2] = { REPORT_ID_1, 0x00 };
+	int ret, wrote;
+
+	ret = hid_int_ep_write(report_1, sizeof(report_1), &wrote);
+
+	SYS_LOG_DBG("Wrote %d bytes with ret %d", wrote, ret);
+
+	/* Increment reported data */
+	report_1[1]++;
+}
+
+static void in_ready_cb(void)
+{
+	k_delayed_work_submit(&delayed_report_send, REPORT_TIMEOUT);
+}
+
+static void status_cb(enum usb_dc_status_code status, u8_t *param)
+{
+	switch (status) {
+	case USB_DC_CONFIGURED:
+		in_ready_cb();
+	default:
```

Comment:
A break is missing there, causing the following line to be printed:
```
[main] [WRN] status_cb: status 3 unhandled
```

Reply:
My bad, fixed now.